src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -382,6 +384,17 @@ private static ILambdaExpression CreateBoundLambdaOperation(BoundLambda boundLam
             return new LazyLambdaExpression(signature, body, isInvalid, syntax, type, constantValue);
         }
 
+        private static ILocalFunctionStatement CreateBoundLocalFunctionStatementOperation(BoundLocalFunctionStatement boundLocalFunctionStatement)
+        {
+            IMethodSymbol localFunctionSymbol = boundLocalFunctionStatement.Symbol;
+            Lazy<IBlockStatement> body = new Lazy<IBlockStatement>(() => (IBlockStatement)Create(boundLocalFunctionStatement.Body));
+            bool isInvalid = boundLocalFunctionStatement.HasErrors;
+            SyntaxNode syntax = boundLocalFunctionStatement.Syntax;
+            ITypeSymbol type = null;
+            Optional<object> constantValue = default(Optional<object>);
```

Comment:
@AlekseyTs Added a note to https://github.com/dotnet/roslyn/issues/19720#issue-230775041 to address this review comment.